.pre-commit-config.yaml

```diff
@@ -31,7 +31,14 @@ repos:
       - id: debug-statements
         language_version: python3
   - repo: https://github.com/asottile/pyupgrade
-    rev: v2.7.3
+    rev: v2.7.4
     hooks:
       - id: pyupgrade
         language_version: python3
+  - repo: git://github.com/luismayta/pre-commit-mypy
+    rev: 0.1.1
+    hooks:
+      - id: mypy
+        args:
+          - --ignore-missing-imports
```

Comment:
that one i'd like to see to gradually go away

ps, can we put all mypy config into setup.cfg?

Reply:
Done